AmmadAmjad-CodeTest/Web/src/AddNewCommentModal/AddNewCommentModal.css

```diff
@@ -0,0 +1,163 @@
+.modal-mask {
+  position: fixed;
+  z-index: 2;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.5);
+  display: table;
+  transition: opacity 0.3s ease;
+}
+
+.hideModal {
+  display: none;
+}
+
+.modal-wrapper {
+  display: table-cell;
+  vertical-align: middle;
+}
+
+.modal-container {
+  width: 450px;
+  margin: 0px auto;
+  background-color: #fff;
+  border-radius: 4px;
+  overflow: hidden;
+  box-shadow: 1px 1px 10px 1px #212121;
+  transition: all 0.3s ease;
+  /* overflow: hidden; */
+}
+
+.modalcontainerbig {
+  width: 90%;
+}
+
+.modal-header h3 {
+  color: black;
+  font-size: 26px;
+  font-family: "Quicksand", sans-serif;
```

Comment:
I don't see where this font is loaded, not sure if it's gonna work on other devices.